Sources/Parsers/ColorsFileParser.swift

```diff
@@ -182,9 +182,11 @@ public final class ColorsXMLFileParser: ColorsFileParser {
   public func parseFile(at path: Path) throws {
     let document = try Fuzi.XMLDocument(string: try path.read())
 
-    for color in document.xpath(XML.colorPath) {
+    for color in document.xpath(XML.colorXPath) {
       let value = color.stringValue
-      guard let name = color["name"], !name.isEmpty else { continue }
+      guard let name = color["name"], !name.isEmpty else {
+        throw ColorsParserError.invalidFile(reason: "Invalid structure, color must have a name.")
```

Comment:
"color \(value) must have a name" to help identify in the error message which line on the XML we're taking about?